### PR TITLE
allow tone frequency of 0

### PR DESCRIPTION
**Pull request by RufusVS**
I left an issue in the issues section, but decided I could simply make a pull request myself.  (I'm pretty new to the process).
This fix allows a specification of a zero frequency to the play_tone function.
Forum: [https://forums.adafruit.com/viewtopic.php?f=60&t=182821](forum)
Issue: [https://github.com/adafruit/Adafruit_CircuitPython_MagTag/issues/67](issue)

### CODE REVIEW

**Comment by TheKitty** [comment]
Why is a tone value of zero desirable? If the user believes a tine might be zero, they could handle.

**Comment by RufusVS** [comment]
> Why is a tone value of zero desirable? If the user believes a tone might be zero, they could handle.

I probably should have quoted my issue or forum post which explained it much better. 
[https://forums.adafruit.com/viewtopic.php?f=60&t=182821](url)

TL;DR
Music often requires rests (periods of silence).  It makes more sense to have it use the same duration intervals that are used for the other notes than handling it as a special case and needing the import of a timer module.  Specifying zero allows for timed periods of silence (rests).

**Comment by makermelissa** [comment]
Have you tested that this works on hardware? If I recall correctly, I think it was causing errors and that's why we limited it to frequency above 0, but it's possible things have changed since then. The play_tone() function is really just a wrapper for the simpleio tone function: https://github.com/adafruit/Adafruit_CircuitPython_SimpleIO/blob/main/simpleio.py#L37 and I'm not sure how it would handle a 0 hertz frequency.

**Review by FoamyGuy — APPROVED** [review]
Looks good to me. 

Tested this successfully on MagTag 7.1.0 beta 1 with code like this:
```py
import time
from adafruit_magtag.magtag import MagTag

magtag = MagTag()

song = [(400, 0.35), (0, 0.5), (467, 0.35), (800, 0.25)]

for note in song:
    magtag.peripherals.play_tone(note[0], note[1])
    time.sleep(note[1])
```
I confirmed there does not seem to be any issue with passing 0 to simpleio and it does result in silent playback.
